s4e-backend/src/main/resources/db/migration/V43__props.sql

```diff
@@ -0,0 +1,5 @@
+-- Create a table to store properties in, for now it's only the granule_path prefix.
```

Comment:
Better name of the `property` table will not be sufficient? What kind of properties will be stored inside this table? AdminProperties? GeneralProperties? GeoSereverProperties? ...

Reply:
For now it's only a place for the granule_path prefix, which is then incorporated in the scene_extended.granule_path, so for now that's just GeoServerProperties. However, a general abstraction of properties fits here without negative implications, so I wouldn't narrow it's usage in the future down to GeoServer related stuff, as something else may emerge.
Also, I have a feeling, that using a whole table for this is an overkill, but I don't know of an alternative key-value store in postgresql. Maybe you know of something like this (key-value, with RW capabilities from SOK), and we could get rid of the table altogether?